src/main/scala/org/kframework/minikore/Constructors.scala

```diff
@@ -0,0 +1,97 @@
+package org.kframework.minikore
+
+import org.kframework.minikore.MiniKore._
+
+
+// Constructor Factory.
```

Comment:
Constructor abstract class to create the kore data structures. The implementation is expected to implement the methods in this abstract class. The advantage of having a constructor interface is that tools (like the debugger), will just need a reference to the constructor's implementation to create new Kore data structures. A default implementation of this class is provided.